lbuild/config.py

```diff
@@ -10,31 +10,35 @@
 # governing this code.
 
 import os
-import anytree
+from os.path import realpath, join
 import pkgutil
 import logging
 import collections
-import lxml.etree
-from os.path import realpath, join
 from pathlib import Path
 
-from .exception import LbuildException
+import lxml.etree
+import anytree
 
-import lbuild.module
+from .exception import LbuildException
 
 LOGGER = logging.getLogger('lbuild.config')
 DEFAULT_CACHE_FOLDER = ".lbuild_cache"
 
+
 class ConfigNode(anytree.AnyNode):
+
     def __init__(self, parent=None):
         anytree.AnyNode.__init__(self, parent, filename=Path())
+
         self._cachefolder = Path()
         self._extends = collections.defaultdict(list)
         self._vcs = []
         self._repositories = []
         self._modules = []
         self._options = {}
 
+        self.filename = None
```

Comment:
I'm not sure if this is correct, the anytree constructor already sets the `self.filename` to `Path()` via its constructor, this may overwrite that!

Reply:
Ok, I'll have to look into the anytree API.